geth/node/txqueue.go

```diff
@@ -1,194 +1,254 @@
 package node
```

Comment:
Diff for this file is messy because I moved the original `txqueue.go` to `txqueue_manager.go` and created `txqueue.go` as a new file.